lib/ansible/modules/system/aix_lvol.py

```diff
@@ -0,0 +1,362 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2016, Alain Dejoux ²adejoux@djouxtech.net>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+
+DOCUMENTATION = '''
+---
+author:
+    - "Alain Dejoux (@adejoux)"
+module: aix_lvol
+short_description: Configure AIX LVM logical volumes
+description:
+  - This module creates, removes or resizes AIX logical volumes. Inspired by lvol module.
+version_added: "2.3"
+options:
+  vg:
+    description:
+    - The volume group this logical volume is part of.
+    required: true
+  lv:
+    description:
+    - The name of the logical volume.
+    required: true
+  lv_type:
+    description:
+    - The type of the logical volume. Default to jfs2.
+    required: false
+  size:
+    description:
+    - The size of the logical volume, size by default in megabytes or optionally with one of the [MGT] units.
+  copies:
+    description:
+    - the number of copies of the logical volume. By default, 1 copy. Maximum copies are 3.
+    required: false
+  policy:
+    choices: [ "maximum", "minimum" ]
+    default: maximum
+    description:
+    - Sets the interphysical volume allocation policy. "maximum" allocates logical partitions across the maximum number of physical volumes.
+      "minimum" allocates logical partitions across the minimum number of physical volumes.
+    required: false
+  state:
+    choices: [ "present", "absent" ]
+    default: present
+    description:
+    - Control if the logical volume exists. If C(present) and the
+      volume does not already exist then the C(size) option is required.
+    required: false
+  opts:
+    description:
+    - Free-form options to be passed to the mklv command
+  pvs:
+    description:
+    - Comma separated list of physical volumes e.g. hdisk1,hdisk2
+    required: false
+'''
+
```

Comment:
These docs should also include a `RETURNS` section so users can see examples of what the module outputs in different scenarios.